File: CarCareHub_/UI_/flutter_desktop/lib/models/klijent.dart
import 'package:json_annotation/json_annotation.dart';
part 'klijent.g.dart';

@JsonSerializable()
class Klijent {
  int klijentId;
  String? ime;
  String? prezime;
  String? username;
  String? email;
  String? password;
  String? passwordAgain;
  String? lozinkaSalt;
  String? lozinkaHash;
  String? spol;
  String? brojTelefona;
  int? gradId;
  int? ulogaId;
bool? vidljivo;
String? adresa;


  Klijent(this.klijentId, this.vidljivo, this.adresa, this.ime,this.prezime,this.username,this.email,this.password,this.lozinkaSalt,this.lozinkaHash,this.spol,this.brojTelefona,this.gradId,this.passwordAgain);

  
  factory Klijent.fromJson(Map<String,dynamic> json) => _$KlijentFromJson(json);


  Map<String,dynamic> toJson() => _$KlijentToJson(this);
}

File: CarCareHub_/CarCareHub.Model/Klijent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CarCareHub.Model
{
    public class Klijent
    {
        public int KlijentId { get; set; }
        public string? Ime { get; set; }
        public string? Prezime { get; set; }
        public string? Username { get; set; }
        public string? Email { get; set; }
        public bool Vidljivo { get; set; }
        public string? Password { get; set; }
        public string? LozinkaSalt { get; set; }
        public string? LozinkaHash { get; set; }
        public string? Spol { get; set; }
        public string? BrojTelefona { get; set; }
        public string? Adresa { get; set; }
        public int? GradId { get; set; }
        public virtual Grad? Grad { get; set; }
        public int UlogaId { get; set; }
        public virtual Uloge uloga { get; set; }
    }
}
